### DISCUSSION TITLE
The element type information storage in the Result Class

### DISCUSSION THREAD

**Discussion by mmgel** [pyansys/pymapdl-reader#131] [Q&A] [answered]
In the Result Class, the element type information are stored in _nodelm_, _nodfor_, _nodstr_ and _keyopts_ as ndarray with 10000 rows.  The index of these arrays are defined by etype_ref, aka the element type reference number in Ansys. If the element type reference number is incontinuous, and larger than 10000, the program will report an IndexError. 
![very large element type number](https://user-images.githubusercontent.com/61673859/129477652-40958dd3-042b-439c-9492-063df4348636.jpg)

Is there any solution to this problem？

**Comment by germa89** [comment, marked as answer]
Pinging @akaszynski for further input.

**Reply by akaszynski** [reply]
> Is there any solution to this problem？

This was fixed in https://github.com/pyansys/pymapdl-reader/pull/32
